libs/Microsoft.MixedReality.WebRTC/Library.cs

```diff
@@ -6,6 +6,23 @@
 
 namespace Microsoft.MixedReality.WebRTC
 {
+    /// <summary>
+    /// Audio device module for Windows Desktop platform.
+    /// </summary>
+    public enum AudioDeviceModule : byte
+    {
+        /// <summary>
+        /// Legacy audio device module (ADM1); not recommended unless there is an issue
+        /// with the new default one.
+        /// </summary>
+        LegacyModule = 1,
+
+        /// <summary>
+        /// New CoreAudio based audio device module (ADM2). This is the default.
+        /// </summary>
+        DefaultModule = 2
```

Comment:
nit: any reason why the enum values don't have the same name here and in the C API?

Reply:
No good ones; I changed my mind when doing the C# projection and thought it was better to be more explicit about what to expect from those (default/legacy) than having some history-related meaningless names (ADM1/ADM2) that do not convey any information for most users. I can change the C API ones too if you prefer.

Reply:
In general I'd prefer explicit naming, what's considered "legacy" and "default" might change over time, but this is not a likely issue in this case, so we can go with these. As long as they match - them being different seems to suggest that they have a reason to be different.

Reply:
Fixed.